### PR TITLE
Various Story edits

### PR DESCRIPTION
**Pull request by LeonisSupreme**
I've been trying to understand the code and have been messing around, so these are my first story edits, and additions, with some comments in some files which don't need to be committed, but I didn't remove.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -25,23 +25,24 @@
...
+'First normal form? What does that mean?',
+'By using a relational database with nonrelational practices we get the worst of both worlds!',
+'I haven\'t paid attention or read any comments, therefore it\'s impossible to understand!',
+'Don\'t use that! Oh, you\'re asking why? Well... just don\'t it.',
+'I don\'t wanna explain, just Google it.',
+'What are cogs?',
+'This is MY project. You\'re just freeloaders.',
+'You\'ve got three weeks to fix EVERYTHING.',
+'No-one agrees? Too bad! My idea it is.',
+'The next version will be written in Java only!'
```
Comment (Randium) [inline main.py]: Why did you change all this?

File: story_time/channel_introduction.py [hunk 2]
```diff
@@ -8,14 +8,14 @@ def channel_introduction(role):
 **WELCOME, ASSASSIN**
 **=========================**
 This town is like an old kitchen... it's a giant mess. Luckily, the town has *you*, \
-their protector, who picks out the cockroaches for them and squishes them like the puny animals they are. 
+their protector, who picks out the cockroaches for them and squishes them like the puny animals they are.
 You do not exactly know what is about to happen, but your senses never lie; \
 something *TERRIBLE* is happening, and it depends on your actions how much the town will suffer from it!
-You do not know your allies, your masters taught you never to trust anyone. However, \
+You do not know your allies- your masters taught you never to trust anyone. However, \
```
Comment (Randium) [inline story_time/channel_introduction.py]: If you want to put a - in-between, you gotta put a space in front.

**Comment by LeonisSupreme** [comment]
Its... a bit messy. All those "merges" are me staying updated from mainstream. Most of the stuff is in story edits and additions.

File: story_time/morning.py [hunk 3]
```diff
@@ -1,39 +1,58 @@
 import random
 
-def story_time(victims):
+def morning(victims): #make consistent to "evening", if there are no dependencies?
```
Comment (Randium) [inline story_time/morning.py]: I don't think there are. But have you checked?

File: main.py (hunk 1 above)
Comment (LeonisSupreme) [reply main.py]: I just thought it would be more consistent, but it doesn't really affect anything

File: story_time/morning.py (hunk 3 above)
Comment (Randium) [reply story_time/morning.py]: Aight. Then that's okay.

File: story_time/channel_introduction.py (hunk 2 above)
Comment (LeonisSupreme) [reply story_time/channel_introduction.py]: Got it. Thanks!
